### PR TITLE
tiup: fix some details in tiup deployment doc

### PR DESCRIPTION
**Pull request by yikeke**
### What is changed, added or deleted? (Required)

fix some details in tiup deployment doc

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

**If you select two or more versions from above**, to trigger the bot to cherry-pick this PR to your desired release version branch(es), you **must** add corresponding labels such as **needs-cherry-pick-4.0**, **needs-cherry-pick-3.1**, **needs-cherry-pick-3.0**, and **needs-cherry-pick-2.1**.

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/2705/
- Other reference link(s):

### CODE REVIEW

File: how-to/deploy/orchestrated/tiup.md [hunk 1]
```diff
@@ -1437,6 +1333,85 @@ tidb_servers:
...
+v3.1.0-beta.2             2020-03-19T00:48:48.266468238+08:00  linux/amd64,darwin/amd64
+v3.1.0-rc                 2020-04-02T23:43:17.456327834+08:00  linux/amd64,darwin/amd64
+v4.0.0-beta               2020-03-13T12:43:55.508190493+08:00  linux/amd64,darwin/amd64
+v4.0.0-beta.1             2020-03-13T12:30:08.913759828+08:00  linux/amd64,darwin/amd64
+v4.0.0-beta.2  YES        2020-03-18T22:52:00.830626492+08:00  linux/amd64,darwin/amd64
+v4.0.0-rc                 2020-04-09T00:10:32+08:00            linux/amd64,darwin/amd64
+nightly                   2020-04-10T08:42:23+08:00            darwin/amd64,linux/amd64
+```
+
+### How to view components and versions supported by TiUP
```
Comment (yikeke) [inline how-to/deploy/orchestrated/tiup.md:1377]:
```suggestion
### How to view the TiDB components supported by TiUP 
```

File: how-to/deploy/orchestrated/tiup.md [hunk 2]
```diff
@@ -1437,6 +1333,85 @@ tidb_servers:
...
+v4.0.0-beta               2020-03-13T12:43:55.508190493+08:00  linux/amd64,darwin/amd64
+v4.0.0-beta.1             2020-03-13T12:30:08.913759828+08:00  linux/amd64,darwin/amd64
+v4.0.0-beta.2  YES        2020-03-18T22:52:00.830626492+08:00  linux/amd64,darwin/amd64
+v4.0.0-rc                 2020-04-09T00:10:32+08:00            linux/amd64,darwin/amd64
+nightly                   2020-04-10T08:42:23+08:00            darwin/amd64,linux/amd64
+```
+
+### How to view components and versions supported by TiUP 
+
+Execute the following command to view the components and versions that TiUP supports managing:
```
Comment (yikeke) [inline how-to/deploy/orchestrated/tiup.md:1379]:
```suggestion
Execute the following command to view the TiDB components that TiUP supports managing:
```

File: how-to/deploy/orchestrated/tiup.md [hunk 3]
```diff
@@ -1437,6 +1333,85 @@ tidb_servers:
...
+v3.0.8                    2020-03-17T14:03:29.575448277+08:00  linux/amd64,darwin/amd64
+v3.0.9                    2020-03-13T13:02:15.947260351+08:00  linux/amd64,darwin/amd64
+v3.0.10                   2020-03-13T14:11:53.774527401+08:00  linux/amd64,darwin/amd64
+v3.0.11                   2020-03-13T15:31:06.94547891+08:00   linux/amd64,darwin/amd64
+v3.0.12        YES        2020-03-20T11:36:28.18950808+08:00   linux/amd64,darwin/amd64
+v3.1.0-beta.2             2020-03-19T00:48:48.266468238+08:00  linux/amd64,darwin/amd64
+v3.1.0-rc                 2020-04-02T23:43:17.456327834+08:00  linux/amd64,darwin/amd64
+v4.0.0-beta               2020-03-13T12:43:55.508190493+08:00  linux/amd64,darwin/amd64
+v4.0.0-beta.1             2020-03-13T12:30:08.913759828+08:00  linux/amd64,darwin/amd64
+v4.0.0-beta.2  YES        2020-03-18T22:52:00.830626492+08:00  linux/amd64,darwin/amd64
```
Comment (superlzs0476) [inline how-to/deploy/orchestrated/tiup.md:1372]:
```suggestion
v4.0.0-beta.2             2020-03-18T22:52:00.830626492+08:00  linux/amd64,darwin/amd64
```

File: how-to/deploy/orchestrated/tiup.md [hunk 4]
```diff
@@ -1437,6 +1333,85 @@ tidb_servers:
...
+v3.0.3                    2020-03-27T22:41:16.279411145+08:00  linux/amd64,darwin/amd64
+v3.0.4                    2020-03-27T22:43:35.362550386+08:00  linux/amd64,darwin/amd64
+v3.0.5                    2020-03-27T22:46:01.016467032+08:00  linux/amd64,darwin/amd64
+v3.0.6                    2020-03-13T11:55:17.941641963+08:00  linux/amd64,darwin/amd64
+v3.0.7                    2020-03-13T12:02:22.538128662+08:00  linux/amd64,darwin/amd64
+v3.0.8                    2020-03-17T14:03:29.575448277+08:00  linux/amd64,darwin/amd64
+v3.0.9                    2020-03-13T13:02:15.947260351+08:00  linux/amd64,darwin/amd64
+v3.0.10                   2020-03-13T14:11:53.774527401+08:00  linux/amd64,darwin/amd64
+v3.0.11                   2020-03-13T15:31:06.94547891+08:00   linux/amd64,darwin/amd64
+v3.0.12        YES        2020-03-20T11:36:28.18950808+08:00   linux/amd64,darwin/amd64
```
Comment (superlzs0476) [inline how-to/deploy/orchestrated/tiup.md:1367]:
```suggestion
v3.0.12                   2020-03-20T11:36:28.18950808+08:00   linux/amd64,darwin/amd64
```

File: how-to/deploy/orchestrated/tiup.md [hunk 5]
```diff
@@ -1437,6 +1333,85 @@ tidb_servers:
...
+v3.1.0-beta.2             2020-03-19T00:48:48.266468238+08:00  linux/amd64,darwin/amd64
+v3.1.0-rc                 2020-04-02T23:43:17.456327834+08:00  linux/amd64,darwin/amd64
+v4.0.0-beta               2020-03-13T12:43:55.508190493+08:00  linux/amd64,darwin/amd64
+v4.0.0-beta.1             2020-03-13T12:30:08.913759828+08:00  linux/amd64,darwin/amd64
+v4.0.0-beta.2  YES        2020-03-18T22:52:00.830626492+08:00  linux/amd64,darwin/amd64
+v4.0.0-rc                 2020-04-09T00:10:32+08:00            linux/amd64,darwin/amd64
```
Comment (superlzs0476) [inline how-to/deploy/orchestrated/tiup.md:1373]:
```suggestion
v4.0.0-rc      YES        2020-04-09T00:10:32+08:00            linux/amd64,darwin/amd64
```

File: how-to/deploy/orchestrated/tiup.md [hunk 6]
```diff
@@ -1437,6 +1333,85 @@ tidb_servers:
...
+- `Installed` is whether or not the component is installed
+- `Platforms` is the supported platform
+- `Description` is the component description
+
+```log
+Available components (Last Modified: 2020-02-27T15:20:35+08:00):
+Name               Installed                                                                                                             Platforms                 Description
+----               ---------                                                                                                             ---------                 -----------
+tidb               YES(v3.0.12,v4.0.0-rc)                                                                                            darwin/amd64,linux/amd64  TiDB is an open source distributed HTAP database compatible with the MySQL protocol
+tikv               YES(v3.0.12,v4.0.0-rc)                                                                                            darwin/amd64,linux/amd64  Distributed transactional key-value database, originally created to complement TiDB
```
Comment (superlzs0476) [inline how-to/deploy/orchestrated/tiup.md:1397]:
```suggestion
tikv               YES(v4.0.0-rc)                                                                                                    darwin/amd64,linux/amd64  Distributed transactional key-value database, originally created to complement TiDB
```

File: how-to/deploy/orchestrated/tiup.md [hunk 7]
```diff
@@ -1437,6 +1333,85 @@ tidb_servers:
...
+pd                 YES(v3.0.12,v4.0.0-rc)                                                                                            darwin/amd64,linux/amd64  PD is the abbreviation for Placement Driver. It is used to manage and schedule the TiKV cluster
```
Comment (superlzs0476) [inline how-to/deploy/orchestrated/tiup.md:1398]:
```suggestion
pd                 YES(v4.0.0-rc)                                                                                                    darwin/amd64,linux/amd64  PD is the abbreviation for Placement Driver. It is used to manage and schedule the TiKV cluster
```

File: how-to/deploy/orchestrated/tiup.md [hunk 8]
```diff
@@ -1437,6 +1333,85 @@ tidb_servers:
...
+- `Name` is the supported component name
+- `Installed` is whether or not the component is installed
+- `Platforms` is the supported platform
+- `Description` is the component description
+
+```log
+Available components (Last Modified: 2020-02-27T15:20:35+08:00):
+Name               Installed                                                                                                             Platforms                 Description
+----               ---------                                                                                                             ---------                 -----------
+tidb               YES(v3.0.12,v4.0.0-rc)                                                                                            darwin/amd64,linux/amd64  TiDB is an open source distributed HTAP database compatible with the MySQL protocol
```
Comment (superlzs0476) [inline how-to/deploy/orchestrated/tiup.md:1396]:
```suggestion
tidb               YES(v4.0.0-rc)                                                                                                    darwin/amd64,linux/amd64  TiDB is an open source distributed HTAP database compatible with the MySQL protocol
```

File: how-to/deploy/orchestrated/tiup.md [hunk 9]
```diff
@@ -47,7 +47,8 @@ The software and hardware recommendations for the **target machines** are as fol
     - Under AMD64 architecture, it is recommended to use CentOS 7.3 or above as the operating system.
     - Under ARM architecture, it is recommended to use CentOS 7.6 1810 as the operating system.
 - For the file system of TiKV data files, it is recommended to use EXT4 format. (refer to [Step 3](#step-3-mount-the-data-disk-ext4-filesystem-with-options-on-the-target-machines-that-deploy-tikv)) You can also use CentOS default XFS format.
-- The target machines can communicate with each other on the Intranet. (It is recommended to disable the firewall `firewalld`, or open the required ports between the nodes of the TiDB cluster.)
+- The target machines can communicate with each other on the Intranet. (It is recommended to [disable the firewall `firewalld`](#how-to-stop-the-firewall-service-of-deployment-machines), or open the required ports between the nodes of the TiDB cluster.)
```
Comment (lilin90) [inline how-to/deploy/orchestrated/tiup.md:50]:
```suggestion
- The target machines can communicate with each other on the Intranet. (It is recommended to [disable the firewall `firewalld`](#how-to-stop-the-firewall-service-of-deployment-machines), or enable the required ports between the nodes of the TiDB cluster.)
```

File: how-to/deploy/orchestrated/tiup.md [hunk 10]
```diff
@@ -330,108 +331,178 @@ cat topology.yaml
...
+# # Global variables are applied to all deployments and as the default value of
+# # them if the specific deployment value missing.
+
 global:
   user: "tidb"
   ssh_port: 22
   deploy_dir: "/tidb-deploy"
   data_dir: "/tidb-data"
 
+# # Monitored variables are used to all the machine
```
Comment (lilin90) [inline how-to/deploy/orchestrated/tiup.md:343]:
```suggestion
# # Monitored variables are used to all the machines.
```

File: how-to/deploy/orchestrated/tiup.md [hunk 11]
```diff
@@ -330,108 +331,178 @@ cat topology.yaml
...
 monitored:
-  deploy_dir: "/tidb-deploy/monitored-9100"
-  data_dir: "/tidb-data/monitored-9100"
-  log_dir: "/tidb-deploy/monitored-9100/log"
+  node_exporter_port: 9100
+  blackbox_exporter_port: 9115
+  # deploy_dir: "/tidb-deploy/monitored-9100"
+  # data_dir: "/tidb-data/monitored-9100"
+  # log_dir: "/tidb-deploy/monitored-9100/log"
+# # Server configs are used to specify the runtime configuration of TiDB components
```
Comment (lilin90) [inline how-to/deploy/orchestrated/tiup.md:350]:
```suggestion
# # Server configs are used to specify the runtime configuration of TiDB components.
```

File: how-to/deploy/orchestrated/tiup.md [hunk 12]
```diff
@@ -330,108 +331,178 @@ cat topology.yaml
...
+# # All configuration items can be found in TiDB docs:
+# # - TiDB: https://pingcap.com/docs/stable/reference/configuration/tidb-server/configuration-file/
+# # - TiKV: https://pingcap.com/docs/stable/reference/configuration/tikv-server/configuration-file/
+# # - PD: https://pingcap.com/docs/stable/reference/configuration/pd-server/configuration-file/
+# # All configuration items use points to represent the hierarchy, e.g:
+# #   readpool.storage.use-unified-pool
+# #           ^       ^
+# # You can overwrite this configuration via instance-level `config` field
```
Comment (lilin90) [inline how-to/deploy/orchestrated/tiup.md:358]:
```suggestion
# # You can overwrite this configuration via the instance-level `config` field.
```

File: how-to/deploy/orchestrated/tiup.md [hunk 13]
```diff
@@ -330,108 +331,178 @@ cat topology.yaml
...
+  # http_port: 8123
+  # flash_service_port: 3930
+  # flash_proxy_port: 20170
+  # flash_proxy_status_port: 20292
+  # metrics_port: 8234
+  # deploy_dir: /tidb-deploy/tiflash-9000
+  # data_dir: /tidb-data/tiflash-9000
+  # log_dir: /tidb-deploy/tiflash-9000/log
+  # numa_node: "0,1"
+  # # Config is used to overwrite the `server_configs.tiflash` values
```
Comment (lilin90) [inline how-to/deploy/orchestrated/tiup.md:444]:
```suggestion
  # # Config is used to overwrite the `server_configs.tiflash` values.
```

File: how-to/deploy/orchestrated/tiup.md [hunk 14]
```diff
@@ -330,108 +331,178 @@ cat topology.yaml
...
+
+# pump_servers:
+#   - host: 10.0.1.17
+#     ssh_port: 22
+#     port: 8250
+#     deploy_dir: "/tidb-deploy/pump-8249"
+#     data_dir: "/tidb-data/pump-8249"
+#     log_dir: "/tidb-deploy/pump-8249/log"
+#     numa_node: "0,1"
+#     # Config is used to overwrite the `server_configs.drainer` values
```
Comment (lilin90) [inline how-to/deploy/orchestrated/tiup.md:460]:
```suggestion
#     # Config is used to overwrite the `server_configs.drainer` values.
```

File: how-to/deploy/orchestrated/tiup.md [hunk 15]
```diff
@@ -330,108 +331,178 @@ cat topology.yaml
...
+#     config:
+#       gc: 7
+#   - host: 10.0.1.18
+#   - host: 10.0.1.19
+# drainer_servers:
+#   - host: 10.0.1.17
+#     port: 8249
+#     data_dir: "/tidb-data/drainer-8249"
+#     # if drainer doesn't have checkpoint, use initial commitTS to initial checkpoint
```
Comment (lilin90) [inline how-to/deploy/orchestrated/tiup.md:469]:
```suggestion
#     # If drainer doesn't have any checkpoint, using initial commitTS to initialize the checkpoint
```

File: how-to/deploy/orchestrated/tiup.md [hunk 16]
```diff
@@ -330,108 +331,178 @@ cat topology.yaml
...
+#     # will get a latest timestamp from pd if setting to be -1 (default -1)
```
Comment (lilin90) [inline how-to/deploy/orchestrated/tiup.md:470]:
```suggestion
#     # will get a latest timestamp from pd if `commit_ts` is set to -1 (default -1).
```

File: how-to/deploy/orchestrated/tiup.md [hunk 17]
```diff
@@ -330,108 +331,178 @@ cat topology.yaml
...
+#     commit_ts: -1
+#     deploy_dir: "/tidb-deploy/drainer-8249"
+#     log_dir: "/tidb-deploy/drainer-8249/log"
+#     numa_node: "0,1"
+#     # Config is used to overwrite the `server_configs.drainer` values
```
Comment (lilin90) [inline how-to/deploy/orchestrated/tiup.md:475]:
```suggestion
#     # Config is used to overwrite the `server_configs.drainer` values.
```

File: how-to/deploy/orchestrated/tiup.md [hunk 18]
```diff
@@ -1437,6 +1333,85 @@ tidb_servers:
     log_dir: "deploy/monitored-9100/log"
     ```
 
+### How to view the TiDB versions supported by TiUP
+
+Execute the following command to view the TiDB versions that TiUP supports managing:
```
Comment (lilin90) [inline how-to/deploy/orchestrated/tiup.md:1338]:
```suggestion
Execute the following command to view the TiDB versions that TiUP supports:
```

File: how-to/deploy/orchestrated/tiup.md [hunk 19]
```diff
@@ -1437,6 +1333,85 @@ tidb_servers:
...
+v4.0.0-beta               2020-03-13T12:43:55.508190493+08:00  linux/amd64,darwin/amd64
+v4.0.0-beta.1             2020-03-13T12:30:08.913759828+08:00  linux/amd64,darwin/amd64
+v4.0.0-beta.2             2020-03-18T22:52:00.830626492+08:00  linux/amd64,darwin/amd64
+v4.0.0-rc      YES        2020-04-09T00:10:32+08:00            linux/amd64,darwin/amd64
+nightly                   2020-04-10T08:42:23+08:00            darwin/amd64,linux/amd64
+```
+
+### How to view the TiDB components supported by TiUP 
+
+Execute the following command to view the TiDB components that TiUP supports managing:
```
Comment (lilin90) [inline how-to/deploy/orchestrated/tiup.md:1379]:
```suggestion
Execute the following command to view the TiDB components that TiUP supports:
```

**Comment by yikeke** [comment]
For those code comment issues, I will ask @lonng to refine the configuration file template and then open another PR to fix the issues in this document. FYI. @lilin90 @superlzs0476

**Comment by yikeke** [comment]
> For those code comment issues, I will ask @lonng to refine the configuration file template and then open another PR to fix the issues in this document. FYI. @lilin90 @superlzs0476

Fixed in [Fix grammar and format for topology.example.yaml by yikeke · Pull Request #298 · pingcap-incubator/tiup-cluster](https://github.com/pingcap-incubator/tiup-cluster/pull/298)

**Comment by yikeke** [comment]
> > For those code comment issues, I will ask @lonng to refine the configuration file template and then open another PR to fix the issues in this document. FYI. @lilin90 @superlzs0476
> 
> The template issues are fixed in [Fix grammar and format for topology.example.yaml by yikeke · pingcap-incubator/tiup-cluster](https://github.com/pingcap-incubator/tiup-cluster/pull/298). I'll open another PR in docs/docs-cn to update the docs.

docs/docs-cn updates:
- https://github.com/pingcap/docs-cn/pull/2846/
- https://github.com/pingcap/docs/pull/2360
FYI @lilin90

**Pull request merged**